test/Interop/Cxx/templates/dependent-types.swift

```diff
@@ -85,4 +84,37 @@ DependentTypesTestSuite.test("Function template methods (static)") {
 }
 #endif // Windows
```

Comment:
Looks like it's crashing or mis-compiling on Windows. Probably the same reason as these are failing. You can just pull down this `endif` for now, since we already have a bug tracking this.

Reply:
Just pull this down past all the added tests.

Reply:
Oh okay cool, sounds good!